### DISCUSSION TITLE
Pushed code changes from VSCode does not show in Github

### DISCUSSION THREAD

**Discussion by JulianaRojasGuillen** [community/community#108511] [Pull Requests]
### Select Topic Area

Bug

### Body

Hello, I have this issue:
1. I have this python file **preprocessing.ipynb** that I want to push from VSCode to Github
2. I have run: git push origin main and the file appears in Github, even the git status is
Your branch is up to date with 'origin/main'.
3. When taking a look to the code of **preprocessing.ipynb** in Github, it does not show my code. I also download this file from Github, and it like an empty file.

Also  the git remote is ok:
git remote -v
origin  https://github.com/JulianaRojasGuillen/facial_emotions.git (fetch)
origin  https://github.com/JulianaRojasGuillen/facial_emotions.git (push)

Have someone face and solve this issue?

**Comment by Shahnawazk7944** [comment]
hey @JulianaRojasGuillen 
It seems like you are pushing code that has not been committed yet. Remember there are three areas of your git repository (your local folder, staging, and commit). Whatever code you push to GitHub, only the code present in the commit area will be pushed to GitHub. Ensure that any changes you made to the file are properly committed to your local Git repository. Use `git status` to verify. If changes are uncommitted, use `git add preprocessing.ipynb` followed by `git commit -m "add any comment here"`.

**Reply by JulianaRojasGuillen** [reply]
Thank you so much !! it worked!! :))

**Reply by Shahnawazk7944** [reply]
You're welcome! I'm glad to hear it worked for you!

**Reply by miapaulita** [reply]
Hi @Shahnawazk7944 
I have the same issue as @JulianaRojasGuillen. I did all except for `git add . ` to make sure I had all the files added.

My terminal response was the same as @JulianaRojasGuillen 

—and I dont see any of the files appear in my github repo.

Can I get help please? 👋

**Reply by Shahnawazk7944** [reply]
Can u please explain your issue in more detail?

If u did `git add .` means now your local changes are moved to staging area(staging means your are telling git to start tracking this file/folder for any changes, please use `git status` to know your changes status).

Just commit the changes `git commit -m "add any comment here"`

And then pushed the changes to GitHub using git push `git push -u origin <branch-name>` (e.g., `git push -u origin main` or `git push -u origin master`)

Hopefully your changes will be reflected to GitHub

**Reply by GerryS02** [reply]
I did all that but still do not see the updated code in my repo. I added the files, commit and then push. I see the commits on GitHub but the files are empty. Why's that?
